### PR TITLE
Actions ref PR 5

### PR DESCRIPTION
**Pull request by suearkinunity**
### Purpose of this PR

More changes to the actions refrences. 

### Links

**Jira:** DOCATT-5849

### Comments to Reviewers

Note that most of the files here have no content changes - I just removed the image from their header, because that was an icon from the ProBuilder Toolbar. 

I also have no idea why the two material files ended up here - perhaps Mod can remove them from the PR?

### CODE REVIEW

File: Content/Resources/Materials/FacePicker.mat [hunk 1]
```diff
@@ -26,6 +26,6 @@ Material:
     m_Ints: []
     m_Floats: []
     m_Colors:
-    - _Tint: {r: 1, g: 1, b: 1, a: 1}
+    - _Tint: {r: 0, g: 0, b: 0, a: 1}
```
Comment (JonMUnity) [inline Content/Resources/Materials/FacePicker.mat:29]: I think you added the changes in `Content/Resources/Materials` by accident.

File: Documentation~/Face_Extrude.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
 ## Extrude Faces Options
 
-These options only apply if you are using the **Extrude Faces** button or the **Ctrl/Cmd+E** shortcut.
+| **Property** |   | **Description** |
+| :--- | --- | --- |
+| **Extrude By** |   | When you extrude more than one face, and those faces share an edge, set how the new faces behave along the shared edge. |
+| ![](images/extrude_face_normals.png) | **Face Normals** | Each new face follows the direction of its original face, and the faces are connected to each other. |
+| ![](images/extrude_vertex_normal.png) | **Vertex Normals** | The edges of the new faces farthest from the joint edge follow the direction of the original faces. However, the faces move toward the shared edge rather than follow the original direction for their whole length. |
+| ![](images/extrude_individual_faces.png) | **Individual Faces** | Each new face follows the direction of its original face, but the faces aren't connected to each other. |
+| **Distance** |   | Distance to extrude the faces(s). Use negative values to inset the face, and define inset. |
```
Comment (JonMUnity) [inline Documentation~/Face_Extrude.md:32]: This table is busted. I think the icons have to go in the same column as the title. For example, 
`|  |   **Face Normals** | Each new face follows the direction of its original face, and the faces are connected to each other. |`

File: Documentation~/Selection_SelectByVertexColor.md [hunk 3]
```diff
@@ -1,19 +1,28 @@
...
-![Select all orange faces on the Mesh](images/Example_SelectByVertexColor.png)
+![On the left, a single orange face is selected. On the right, all orange faces are selected.](images/Example_SelectByVertexColor.png)
 
-This action is useful for grouping out sections of your Mesh with different vertex colors. It is available in the [vertex, edge, and face modes](modes.md).
+To select faces by material:
 
-From the main menu, select **Tools** > **ProBuilder** > **Selection** > **Select Vertex Color**.
+1. In the **Tools** overlay, select the **ProBuilder** context.
+1. In the **Tool Settings** overlay, select an edit mode. All three modes support this action.
+1. Select an element. To select more than one color, hold **Shift** and select more elements. <!--this isn't as simple as I make it sound - when you select an edge or a vertex, it's not obvious which face it will reference for the color - it's honestly easier to use only Face mode-->
```
Comment (JonMUnity) [inline Documentation~/Selection_SelectByVertexColor.md:13]: We need to remove the code comments. They display in the published source of the output.

Comment (suearkinunity) [reply Documentation~/Selection_SelectByVertexColor.md:13]: I don't remove comments until right before we merge, unless they're dealt with.

File: Content/Resources/Materials/FacePicker.mat (hunk 1 above)
Comment (suearkinunity) [reply Content/Resources/Materials/FacePicker.mat:29]:
```suggestion
    - _Tint: {r: 1, g: 1, b: 1, a: 1}
```

**Comment by suearkinunity** [comment]
Merging into my *other* PR because frankly this is confusing me.

**Pull request merged**